microsoft-dynamics-crm/3.2/modules/ROOT/pages/index.adoc

```diff
@@ -19,7 +19,12 @@ To use this connector, you must be familiar with:
 
 Before creating an app, you must have login credentials for Microsoft Dynamics CRM SOAP Organization Service to test your connection to your target resource.
 
-Note that you must format a date in milliseconds with a maximum 3 of digits to the right of the decimal.
+Any created or expired date in the web service header must respect the constraints specified the BasicSecurity Profile validation for Web Services, http://www.ws-i.org/Profiles/BasicSecurityProfile-1.0.html, as summarized below:
```

Comment:
Any created or expired date in the web service header created by the Microsoft CRM instance must respect the constraints specified the BasicSecurity Profile validation for Web Services, http://www.ws-i.org/Profiles/BasicSecurityProfile-1.0.html, as summarized below: